pkg/mobile/component.go

```diff
@@ -146,6 +152,16 @@ func (c *component) GetUserInformation(ctx context.Context) (api.UserInformation
 			c.logger.Debug(ctx, "msg", "User is not allowed to access video identification", "id", ctx.Value(cs.CtContextUserID))
 			delete(availableChecks, actionIDNow)
 		}
+
+		// if vouchers are required to proceed to a video identification, check that the balance is bigger than 1
+		if realmAdminConfig.VideoIdentificationAccountingEnabled != nil && realmAdminConfig.VideoIdentificationPrepaymentRequired != nil &&
```

Comment:
No need to enter this `if` statement if `actionIDNow` is not in `availableChecks` (avoid 1 HTTP query and associated SQL query behind it)
You also can write a function to be able to write `realmAdminConfig.IsIDNowActivated()` and hide this long checks

Reply:
Good catch !